src/App.js

```diff
@@ -261,7 +266,7 @@ class App extends Component {
             )}
             {renderAppBar}
             {renderAppBanner}
```

Comment:
Directly use ScrollToTop here, instead of adding it in the function.

Reply:
Done...! Please merge, now!